__tests__/requests.test.js

```diff
@@ -0,0 +1,32 @@
+const seed = require("../db/seeds/seed");
+const testData = require("../db/data/test-data/index");
+const db = require("../db/connection");
+const request = require("supertest");
+const app = require("../db/app");
+beforeEach(() => seed(testData));
+
+afterAll(() => db.end());
+describe("getCategories", () => {
+  it("GET - status: 200 - responds with all categories, including slug and description", () => {
+    return request(app)
+      .get("/api/categories")
+      .expect(200)
+      .then((res) => {
+        expect(res.body.categories.length).toBe(4);
+        res.body.categories.forEach((category) => {
+          expect(typeof category.slug).toBe("string");
+          expect(typeof category.description).toBe("string");
+        });
+      });
+  });
+  it("should return 404 if incorrect endpoint", () => {
```

Comment:
Just a small thing but this test is not relevant to the endpoint describe block it is in. Ideally should be in a more appropriate describe block, it may be on it's own there but at least it's not where it shouldn't be! 😄